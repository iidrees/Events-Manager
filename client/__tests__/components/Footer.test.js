import React from 'react';
import { Provider } from 'react-redux';
import { shallow, mount, render } from 'enzyme';

import sinon from 'sinon';

import Footer from '../../src/components/Footer';

describe('Landing component', () => {
  it('should render without throwing an error', done => {
    expect(shallow(<Footer />).length).toEqual(1);
    expect(<Footer />).toMatchSnapshot();
    done();
  });
});
