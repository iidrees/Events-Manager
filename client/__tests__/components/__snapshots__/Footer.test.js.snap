// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing component should render without throwing an error 1`] = `<Unknown />`;
